build.gradle

```diff
@@ -17,10 +17,10 @@
  */
 
 buildscript {
-  ext.kotlin_version = '1.7.20'
-  ext.compose_compiler_version = '1.3.2'
+  ext.kotlin_version = '1.8.0'
+  ext.compose_compiler_version = '1.4.0'
   ext.detekt_version = '1.22.0'
-  ext.dokka = "org.jetbrains.dokka:dokka-gradle-plugin:$kotlin_version"
+  ext.dokka = "org.jetbrains.dokka:dokka-gradle-plugin:1.7.20"
```

Comment:
looks like dokka is only somewhat related to the kotlin releases. there's no 1.8 version out there yet